src/app/cluster/duck/sagas.ts

```diff
@@ -327,7 +328,18 @@ function* updateClusterRequest(action) {
   const currentCABundle = currentCluster.MigCluster.spec.caBundle;
   const caBundleUpdated = clusterValues.caBundle !== currentCABundle;
 
-  if (!urlUpdated && !tokenUpdated && !azureUpdated && !requireSSLUpdated && !caBundleUpdated) {
+  const currentExposedRegistryPath = currentCluster.MigCluster.spec.exposedRegistryPath;
```

Comment:
Just curious, what happens when `exposedRegistryPath` is undefined in the cluster spec

Reply:
It's a string value, so not defining it should be equivalent to defining it to be an empty string. It is expected to be empty for any cluster where there is no exposed path -- the host cluster in most cases, since for the host cluster we can use the internal path, and remote clusters where the user isn't using direct image migration, since there's no need to expose the registry in that case.

As part of MigPlan src/dest cluster validation, if the plan is using direct image migration and there is no registry path available (internal path for host cluster, exposed registry path otherwise), a critical condition is added to the migplan.